docs/user/cli-reference.md

```
@@ -1178,19 +1178,19 @@ AICR includes a component-driven validation system that automatically checks bun
 
 **Validation Warnings:**
 
-When generating bundles with skyhook-customizations enabled, validation warnings are displayed for missing configuration:
+When generating bundles with nodewright-customizations enabled, validation warnings are displayed for missing configuration:
 
-1. **Workload Selector Warning**: When skyhook-customizations is enabled with training intent, if `--workload-selector` is not set, a warning will be displayed:
+1. **Workload Selector Warning**: When nodewright-customizations is enabled with training intent, if `--workload-selector` is not set, a warning will be displayed:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)